AF + threat vault - auto key (#12169)
demisto/content: Packs/Base/ReleaseNotes/1_9_0.md

#### Scripts
##### CommonServerPython

> - Added support for getting the AutoFocus API Key(server version 6.2.0+).

Reviewer: ```suggestion
- Added support to enable usage of the AutoFocus API key (XSOAR server version 6.2.0+).
```
